integrated_channels/canvas/exporters/content_metadata.py

```diff
@@ -0,0 +1,56 @@
+"""
+Content metadata exporter for Canvas
+"""
+
+from logging import getLogger
+
+from integrated_channels.integrated_channel.exporters.content_metadata import ContentMetadataExporter
+
+LOGGER = getLogger(__name__)
+
+
+class CanvasContentMetadataExporter(ContentMetadataExporter):
+    """
+        Canvas implementation of ContentMetadataExporter.
+    """
+    DATA_TRANSFORM_MAPPING = {
+        'name': 'title',
+        'course_code': 'number',
+        'start_at': 'start',
+        'end_at': 'end',
+        'integration_id': 'key',
+        'syllabus_body': 'description',
+        'default_view': 'default_view',
+    }
+
+    LONG_STRING_LIMIT = 2000
+
+    def transform_description(self, content_metadata_item):
+        """
+        Return the course description and enrollment url as Canvas' syllabus body attribute.
+        This will display in the Syllabus tab in Canvas.
+        """
+        enrollment_url = content_metadata_item.get('enrollment_url', None)
+        base_description = "<a href={enrollment_url}>To edX Course Page</a><br />".format(
```

Comment:
I think this is okay for now, we should ask with Ken what exactly the text etc wants here ... I am good with this for now, we can also review this during team level demo if we do one of those

Reply:
Yep. Absolutely did not write this on the basis this was the final language lol.